datadog/tests/resource_datadog_security_monitoring_default_rule_test.go

```diff
@@ -18,20 +18,22 @@ func TestAccDatadogSecurityMonitoringDefaultRule_Basic(t *testing.T) {
 		PreCheck:          func() { testAccPreCheck(t) },
 		ProviderFactories: accProviders,
 		Steps: []resource.TestStep{
+			// Define an existing default rule as one we want to import
 			{
-				Config: testAccCheckDatadogSecurityMonitoringDefaultDatasource(),
+				Config: testAccDatadogSecurityMonitoringDefaultDatasource(),
 			},
+			// Import the rule
 			{
-				Config:            testAccCheckDatadogSecurityMonitoringDefaultNoop(),
-				ResourceName:      tfSecurityDefaultRuleName,
-				ImportState:       true,
-				ImportStateIdFunc: idFromDatasource,
+				Config:             testAccCheckDatadogSecurityMonitoringDefaultNoop(),
+				ResourceName:       tfSecurityDefaultRuleName,
+				ImportState:        true,
+				ImportStateIdFunc:  idFromDatasource,
+				ImportStatePersist: true,
```

Comment:
We need to persist the state so that we can access it in the next step. No checks are run if we're importing a state in a test step.